[DOCS] correct the default value of ignore_throttled (#60036)
elastic/elasticsearch: docs/reference/search/search-template.asciidoc

`ignore_throttled`::

>   (Optional, boolean) If `true`, specified concrete, expanded or aliased indices 
>   are not included in the response when throttled. Defaults to `true`.

Reviewer: ```suggestion
(Optional, boolean) If `true`, concrete, expanded or aliased indices are
ignored when frozen. Defaults to `true`.
```
